src/ensembl/src/content/app/browser/track-panel/track-panel-modal/TrackPanelModal.tsx

```diff
@@ -65,9 +65,9 @@ export const TrackPanelModal = (props: TrackPanelModalProps) => {
   };
   return (
     <section className={styles.trackPanelModal}>
-      <button onClick={onClickHandler} className={styles.closeButton}>
-        <img src={closeIcon} alt="Close track panel modal" />
-      </button>
+      <div className={styles.closeButton}>
+        <CloseButton onClick={onClickHandler} />
```

Comment:
`onClickHandler ` -> either `onClick` or `clickHandler`

Reply:
That's done